assets/openobserve/main.py

```
@@ -0,0 +1,96 @@
+    def __init__(self, dataset):
+        super().__init__(dataset)
+
+    @property
+    def compressed_size(self):
+        """
+        Returns the size of the compressed dataset
+        """
+        #self.terminate()
+        compressed_size = self.get_disk_usage(f"{OPENOBSERVE_DATA_DIR}/openobserve/stream/files/default")
+        #self.launch()
+        return compressed_size
```

Ruff (0.12.2)

Unnecessary assignment to `compressed_size` before `return` statement

Remove unnecessary assignment

(RET504)